### ISSUE TITLE
Dataset and performance

### ISSUE THREAD

**Issue by wyk0517** [zhengyi123456/Pytorch_Deep_Image_Matting#1]
Hi， do you have the dataset paper mentioned? 
what's more, How about your implement's performance? can you  get the author's results?

**Comment by zhengyi123456** [comment]
Hi, If you want to data, you can send the e-mail about the request of the dataset to the second author of the paper, he will send you the train and eval dataset, including alpha and fg image. I do this project for a school class, so I did not fine-tune it or try to get the best model. However, just according to my personal judgement, the model did not perform very well as the paper said. you can still try it, the detail of model mentioned in the segnet.py. and I also upload an input image and its output, maybe you can feel the performance by your own. If you still have any problem, please contact with me. Thanks

**Comment by wyk0517** [comment]
> Hi, If you want to data, you can send the e-mail about the request of the dataset to the second author of the paper, he will send you the train and eval dataset, including alpha and fg image. I do this project for a school class, so I did not fine-tune it or try to get the best model. However, just according to my personal judgement, the model did not perform very well as the paper said. you can still try it, the detail of model mentioned in the segnet.py. and I also upload an input image and its output, maybe you can feel the performance by your own. If you still have any problem, please contact with me. Thanks

Thank you, you are very nice! I will read your code. Are you a student in china? maybe we can make friends. hahah

**Comment by zhengyi123456** [comment]
Yes, I am a postgraduate student in shanghai. It will be my honor to be friends with you. However, I am not very experienced in deep learning coding and I have not attached any readme file with the code. Therefore, if you have any problems about the code, please let me know and I will response as soon as possible.

**Comment by wyk0517** [comment]
> Yes, I am a postgraduate student in shanghai. It will be my honor to be friends with you. However, I am not very experienced in deep learning coding and I have not attached any readme file with the code. Therefore, if you have any problems about the code, please let me know and I will response as soon as possible.

我明年也在上海读研， 学长哪个学校呀， 我刚学深度学习不久， 还没自己复现过paper，学习一下学长的思路

**Comment by zhengyi123456** [comment]
你好，我是复旦大学的研究生。如果是其他关于深度学习或者是复现的问题，我强烈建议我们通过邮件的形式交流，祝好~
